src/utils/_variables.scss

```diff
@@ -0,0 +1,11 @@
+$cell-size: 100px;
+
+$start-day: (
+  1 : mon,
+  2 : tue,
+  3 : wed,
+  4 : thu,
+  5 : fri,
+  6 : sat,
+  7 : sun,
+);
```

Comment:
you have this utils folder 2 times delete 1 
<img width="374" alt="image" src="https://user-images.githubusercontent.com/100297341/204282740-d6218679-4f5d-4c42-97ff-fb2be7d302a8.png">